src/libs/vwidgets/vmaingraphicsview.cpp

```diff
@@ -714,12 +709,14 @@ void VMainGraphicsView::mouseMoveEvent(QMouseEvent *event)
 {
     if ( (dragMode() == QGraphicsView::ScrollHandDrag) ||  isPanDragActive  )
     {
-        QScrollBar *hBar = horizontalScrollBar();
-        QScrollBar *vBar = verticalScrollBar();
+        QScrollBar *horz = horizontalScrollBar();
+        QScrollBar *vert = verticalScrollBar();
         const QPoint delta = event->pos() - m_ptStartPos;
-        hBar->setValue(hBar->value() + (isRightToLeft() ? delta.x() : -delta.x()));
-        vBar->setValue(vBar->value() - delta.y());
+        horz->setValue(horz->value() + (isRightToLeft() ? delta.x() : -delta.x()));
+        vert->setValue(vert->value() - delta.y());
         m_ptStartPos = event->pos();
+        unsigned long delay = static_cast<unsigned long>(200 + (200 - qApp->Settings()->getAutoScrollSpeed()));
+        QThread::msleep(10000); // delay is the inverse of the speed
```

Comment:
This method is called when the view is panned using the middle mouse button, and this `msleep(10000)` is making the whole software freeze for 10s each time the middle mouse button is pressed.

Reply:
My bad... I was testing the "delay" and put a ridicoulous number in, and I forgot to remove these lines. Thing is, and I think I've mentioned this before - I never use the pan to scroll so I obvioulsy failed to catch this. DOH.

Reply:
Yes 😊